From e4f0db3bb465f49bef478c050abbee9a86fa69bd Mon Sep 17 00:00:00 2001
From: Deepak Sharma <Deepak.Sharma@amd.com>
Date: Thu, 11 Aug 2016 15:05:10 +0530
Subject: [PATCH 2/2] Remove egl+drm dpendency error while building radeonsi
 driver

Error check is not required as this was due to
glamor dependency for 2d acclearation.

Change-Id: Ib055f69f2a78cb23a3238917b5003554c2344db5
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index c492e15..faaa3ae 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2376,7 +2376,6 @@ if test -n "$with_gallium_drivers"; then
             gallium_require_drm "radeonsi"
             gallium_require_drm_loader
             radeon_llvm_check "radeonsi"
-            require_egl_drm "radeonsi"
             ;;
         xnouveau)
             HAVE_GALLIUM_NOUVEAU=yes
-- 
1.9.1

